.github/workflows/workflow.yml

```diff
@@ -0,0 +1,71 @@
+name: Workflow CI
+
+on:
+  workflow_run:
+    workflows:
+      - "Pull Request CI"
+    types:
+      - completed
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.1
+        with:
+          persist-credentials: false
+      - run: |
+          PR=$(curl https://api.github.com/search/issues?q=${{ github.event.workflow_run.head_sha }} |
+          grep -Po "(?<=${{ github.event.workflow_run.repository.full_name }}\/pulls\/)\d*" | head -1)
+          echo "PR=$PR" >> $GITHUB_ENV
+      - run: |
+          git fetch origin ${{ github.event.workflow_run.head_sha }}
+          git checkout ${{ github.event.workflow_run.head_sha }}
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: "15.x"
+      - run: npm ci
+      - run: |
+          npm test
+          sed -i '/out\//d' .gitignore
+          sed -i '/out-wpt\//d' .gitignore
```

Comment:
unrelated: I think these two lines are obsolete now that we don't upload by git push to gh-pages?